libs/blocks/merch-card/merch-card.js

```diff
@@ -111,13 +111,22 @@ const MULTI_OFFER_CARDS = ['plans', 'product', MINI_COMPARE_CHART];
 // Force cards to refresh once they become visible so that the footer rows are properly aligned.
 const intersectionObserver = new IntersectionObserver((entries) => {
   entries.forEach((entry) => {
-    const container = entry.target.closest('main > div');
+    const container = entry.target.closest('main > div:not([data-status])');
     if (!container) return;
     [...container.querySelectorAll('merch-card')].forEach((card) => card.requestUpdate());
     intersectionObserver.unobserve(entry.target);
   });
 });
 
+const addTabClickListener = (container) => {
```

Comment:
I don't think adding an event listener is needed.
A merch card that is inside a tab that is hidden is fine,  the intersectionObserver should allow us to detect when the merch-card inside the tab becomes visible when the tab is activated.

Reply:
I suggest to simplify as follows, feel free to merge proposed change.

https://github.com/Axelcureno/milo/pull/6

test: https://main--cc--adobecom.hlx.page/cc-shared/fragments/merch/products/illustrator/mini-compare/segment-blade?milolibs=mwpw-142003-test--milo--yesil

Reply:
Correct, event listener removed. Thank you.